### ISSUE TITLE
Specifying multiple comma separated values does not work with cb

### ISSUE THREAD

**Issue by el8** [ibmcb/cbtool#375]
Trying to do something like:
./cb cldalter vm_templates fio=size:32gb,imageids:1,imageid1:00726830
results in an error:

 ############################# Executing command "imageids:1" (specified on the command line) 

*** Unknown syntax: imageids:1

 ############################# Executing command "imageid1:15101" (specified on the command line) 

*** Unknown syntax: imageid1:15101

while calling the identical command after entering ./cb works fine.

**Comment by maugustosilva** [comment]
you're right, this is a bug. Right now, we are using comma to separate between multiple "initial commands" when `cb` is invoked. I need to think of a better solution for it. Thanks for the report.
